### PR TITLE
Fix Replays with CP selected

### PR DESCRIPTION
**Pull request by shipyy**
error occuring before 
image

was due to wrong query format resulting in a false positive regarding duplicate entry

added a check to not display the starting points from replay incase there was no replay cp ticks found when loading

also swapped multiple queries to a unique transaction instead

### CODE REVIEW

**Comment by tslashd** [comment]
Error seems to not be present after this.

_Before this fix the replay from the map mentioned in PR has been replaced by a non WR tho_

File: addons/sourcemod/scripting/surftimer/sql.sp [hunk 1]
```diff
@@ -3427,28 +3427,33 @@ public void db_UpdateReplaysTick(int client, int style){
...
 	}
 	else{
 		for(int i = 0; i < cp_count; i++){
 			Format(szQuery, sizeof(szQuery), sql_updateReplayCPTicks, g_szMapName, i+1, g_iCPStartFrame[style][i], style);
-			SQL_TQuery(g_hDb, SQL_UpdateReplaysTickCallback, szQuery, _, DBPrio_Low);
+			TicksTransaction.AddQuery(szQuery);
 		}
 	}
+
+	SQL_ExecuteTransaction(g_hDb, TicksTransaction, db_TicksTransactionOnSuccess, db_TicksTransactionOnFailure, DBPrio_Low);
```
Comment (Bara) [inline addons/sourcemod/scripting/surftimer/sql.sp:3446]: DBPrio_Low is here wrong. Replace it with this:
`SQL_ExecuteTransaction(g_hDb, TicksTransaction, db_TicksTransactionOnSuccess, db_TicksTransactionOnFailure, _, DBPrio_Low);`

**Review by Bara — CHANGES_REQUESTED** [review]

File: addons/sourcemod/scripting/surftimer/sql.sp (hunk 1 above)
Comment (Bara) [reply addons/sourcemod/scripting/surftimer/sql.sp:3446]: or `.priority=DBPrio_Low` works too

**Comment by tslashd** [comment]
does not update CPs in chat until reconnecting to the server

**Comment by shipyy** [comment]
the errors related to ck_checkpoints were happening because the changes were not merged from the other PR(#499 ) that fixes taht issue

**Review by Bara — APPROVED** [review]

**Pull request merged**